llama_stack/providers/utils/sqlstore/authorized_sqlstore.py

```diff
@@ -59,7 +60,8 @@ def __init__(self, sql_store: SqlStore):
 
         :param sql_store: Base SqlStore implementation to wrap
         """
-        self.sql_store = sql_store
+        self.sql_store: SqlStore = sql_store
+        self.supported_databases: list[SqlStoreType] = list(SqlStoreType)
```

Comment:
I don't think you should ever use instance variables for what are essentially constants. Also you could probably make this into an UnsupportedSqlStoreError() and not need to pass the list from outside.